### PR TITLE
[IMP] account : new report 'duplicata' for an invoice

### PR DESCRIPTION
**Pull request by tivisse**
TASK : https://www.odoo.com/web#id=13390&view_type=form&model=project.task&action=333&active_id=248
PAD : https://pad.odoo.com/p/r.WjhSt00fB6Je093h

### CODE REVIEW

File: addons/account/views/account_report.xml [hunk 1]
```diff
@@ -13,5 +13,15 @@
             attachment="(object.state in ('open','paid')) and ('INV'+(object.number or '').replace('/','')+'.pdf')"
         />
 
+        <report 
+            id="account_duplicates"
```
Comment (tde-banana-odoo) [inline addons/account/views/account_report.xml]: id: account_invoice_report_duplicate

File: addons/account/views/report_invoice.xml [hunk 2]
```diff
@@ -157,5 +157,22 @@
         </t>
     </t>
 </template>
+
+<template id="report_invoice_duplicate" inherit_id="account.report_invoice_document" primary="True">
```
Comment (tde-banana-odoo) [inline addons/account/views/report_invoice.xml]: account_invoice_report_duplicate ?

File: addons/account/views/report_invoice.xml [hunk 3]
```diff
@@ -157,5 +157,22 @@
...
+    <xpath expr="//h2" position="replace">
+        <h2>
+            <span>Duplicate</span>
+            <span t-field="o.number"/>
+        </h2>
+    </xpath>
+</template>
+
+<template id="report_duplicate">
```
Comment (tde-banana-odoo) [inline addons/account/views/report_invoice.xml]: account_invoice_report_duplicate_main ?

**Comment by tivisse** [comment]
Landed in odoo/master https://github.com/odoo/odoo/commit/93bdbb79048a3745682af2d9b1433eca1db3376f

**Comment by antonylesuisse** [comment]
This seems wrong to me translate_doc is deprecated as we now have a cleaner system to switch translation when calling a qweb template. We should convert all translate_doc to the new syntax before v9 @sle-odoo Am I wrong ?

**Comment by antonylesuisse** [comment]
I found it in sales <t t-call="sale.report_saleorder_document" t-lang="doc.partner_id.lang"/>

**Comment by atp-odoo** [comment]
@antonylesuisse :+1:

**Comment by tivisse** [comment]
@antonylesuisse I didn't found it. Also, we want a invoice duplicate, not a sale order one. Are you sure it's an invoice duplicated ?

**Comment by antonylesuisse** [comment]
Change ligne

<t t-raw="translate_doc(doc_id, doc_model, 'partner_id.lang', 'account.account_invoice_report_duplicate')"/>

by

<t t-call="'account.account_invoice_report_duplicate" t-lang="doc.partner_id.lang"/>

and grep for remaining deprecated "translate_doc"

**Comment by tivisse** [comment]
OK, will do it in another task.

**Pull request merged**